### ISSUE TITLE
Provide full documentation of custom tags using 2-way binding, binding to args or props etc.

### ISSUE THREAD

**Issue by badiezadegan** [BorisMoore/jsviews#198]
My problem [in this issue](https://github.com/BorisMoore/jsviews/issues/197) is solved by moving the code for assigning the default value to the `init` function of the tag. But Now I have another problem. If I bind an object property to a tag property, whenever the **object property** changes, the **tag property** changes too. But I can't makes it work for the reverse case. When I change the **tag property** manually, the bound **property of the object** does not change. Here is my code:

Here if one changes the `~root.title`, the `value` property of `test` will change.
But In `$('#editTitle').click`, if the commented line get uncommented and its below line, (`$.observable(root).setProperty('title', prompt())`) get commented, then the `~root.title` will not change when one changes the `value`property of `test` tag. 
Thank you very much for answering my questions.

``` html
<!DOCTYPE html>
<html>
<head>
    <title></title>
    <script src="js/jquery-1.9.1.min.js" type="text/javascript"></script>
    <script src="js/jsrender.js" type="text/javascript"></script>
    <script src="js/jquery.observable.js" type="text/javascript"></script>
    <script src="js/jquery.views.js" type="text/javascript"></script>

    <script id="testTemplate" type="text/x-jsrender">
        <div>{^{>value}}</div>
    </script>

    <script id="myTemplate" type="text/x-jsrender">
        <table>
            <tbody>
            <tr>
                <td colspan="0">{^{test value=~root.title /}}</td>
            </tr>
            <tr>
                <td>First Name:</td>
                <td><input type="text" data-link="~root.firstName"/></td>
            </tr>
            <tr>
                <td>Last Name:</td>
                <td><input type="text" data-link="~root.lastName"/></td>
            </tr>
            </tbody>
        </table>
    </script>

    <script type="text/javascript">
        $.views.tags({
            test: {
                init: function () {
                    this.tagCtx.props.value = 'Default Title';

                },
                render: function () {
                    return $.templates('#testTemplate').render(this.tagCtx.props, this.tagCtx, this.tagCtx.view);
                }
            }
        });

        $.templates({myTemplate: "#myTemplate",
            test: "#testTemplate"
        });

        $(function () {
            var root = { firstName: 'John', lastName: 'King'};
            $.link.myTemplate('#container', root);

            $('#editTitle').click(function () {
//                $.observable($.view('#container table td:first div').data).setProperty('value', prompt());
                $.observable(root).setProperty('title', prompt());
                alert(JSON.stringify(root));
            });
        });
    </script>
</head>
<body>
<span id="editTitle">Edit Title</span>
<div id="container"></div>
</body>
</html>
```

**Comment by BorisMoore** [comment]
Two-way binding docs are now complete: http://www.jsviews.com/index.html#link2way

**Comment by BorisMoore** [comment]
This has been fixed in [release v0.9.91](https://github.com/BorisMoore/jsviews.com/commit/f59020fca27090c7e1d948fc82101ad99d662958). There is a lot of new documentation including under this [tag controls](https://www.jsviews.com/#jsvtagcontrols) topic:
